test/mockappserver.hpp

```diff
@@ -62,7 +62,7 @@ class MockAppServerTsxHelper : public AppServerTsxHelper
   MOCK_METHOD0(original_request, pjsip_msg*());
   MOCK_CONST_METHOD0(route_hdr, const pjsip_route_hdr*());
   MOCK_METHOD1(add_to_dialog, void(const std::string&));
-  MOCK_METHOD1(clone_request, pjsip_msg*(pjsip_msg*));
+  MOCK_METHOD1(clone_msg, pjsip_msg*(pjsip_msg*));
```

Comment:
Hmmm. This makes me thing that the pointer of the `AppServerTsxHelper` is to make unit testing possible. In which case it's useful for the `AppServerTsx` and the helper to have exactly the same API (and we shouldn't map from `clone_request` to `clone_msg` in the helper).